### DISCUSSION TITLE
Feature request: Notification

### DISCUSSION THREAD

**Discussion by gab12** [amugofjava/anytime_podcast_player#79] [Ideas]
Hello,
I really like AnyTime, thank you for your application.
I subscribe to a lot of postcasts. It is very difficult to know when a new postcast is available. I have to go to each feed to find out if there's a new postcast. It's tedious.
I therefore agree with the proposal that could answer the problems: [Latest podcasts and other suggestion #48](https://github.com/amugofjava/anytime_podcast_player/issues/48)
But the best would be to receive a notification when a new postcast is available. (a background synchronization should probably be necessary). It would allow me not to miss any postcast.

Thank you

**Comment by amugofjava** [comment]
Hi,

Thank you for your feature request. I completely agree that the current solution of visiting each feed to see if there are new episodes is not ideal and background syncing is needed. It is on the roadmap and if you have any further suggestions or comments on priorities please take a look at the discussion [here](https://github.com/amugofjava/anytime_podcast_player/discussions/72).

Notifications would be very useful and I think these should be configurable on a per-podcast basis so that you only get notifications for the podcasts you really want to know about.
